final_prep/mod_zero_hero.rb

```diff
@@ -1,84 +1,99 @@
-#   - danger_level should be an integer
-#   - save_the_day should be a string a hero would say once they save the day 
-#   - bad_excuse should be a string a hero would say if they are too afraid of the danger_level
-
-# Your function should include an if/else statement that meets the following criteria
-#   - Danger levels that are above 50 are too scary for your hero. Any danger level that is above 50 should result in printing the bad_excuse to the terminal
-#   - Anything danger_level that is between 10 and 50 should result in printing the save_the_day string to the terminal
-#   - If the danger_level is below 10, it means it is not worth your time and should result in printing the string "Meh. Hard pass." to the terminal.
-
-#Test Cases
-announcement = 'Never fear, the Courageous Curly Bracket is here!'
-excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.'
-# assess_situation(99, announcement, excuse) > Should print - 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.'
-#assess_situation(21, announcement, excuse) > should print - 'Never fear, the Courageous Curly Bracket is here!'
-#assess_situation(3, announcement, excuse) > should print - "Meh. Hard pass."
-
-# Declare a new variable - scary_monster - assigned to an hash with the following key/values
-#   - name (string)
-#   - smell (string)
-#   - weight (integer)
-#   - citiesDestroyed (array)
-#   - luckyNumbers (array)
-#   - address (hash with following key/values: number , street , state, zip)
-
-
-# Create a new class called SuperHero
-# - Your class should have the following DYNAMIC values
-#   - name 
-#   - super_power
-#   - age 
-# - Your class should have the following STATIC values
-#   - arch_nemesis, assigned to "The Syntax Error"
-#   - power_level = 100
-#   - energy_level = 50 
-
-# - Create the following class methods
-#   - say_name, should print the hero's name to the terminal
-#   - maximize_energy, should update the energy_level to 1000
-#   - gain_power, should take an argument of a number and INCREASE the power_level by that number
-
-# - Create 2 instances of your SuperHero class
-
-
-# Reflection
+hero_name = "Radioactive Man"
+special_ability = "upending atoms"
+greeting = "Behold! I am #{hero_name}."
+catchphrase = "Up and at them, #{special_ability}!"
+power = 3
+energy = 7
+full_power = power * 500
+full_energy = energy + 150
+is_human = true
+identity_concealed = false
+arch_enemies = ["King Kong", "James Joyce", "Studio Executive"]
+sidekicks = ["Fallout Boy", "Jolly Old St Nick", "Mickey Rooney"]
+arch_enemies.push("Johnny Rocket")
+puts arch_enemies
+sidekicks.shift
+puts sidekicks
+
+def asses_situation(danger_level, save_the_day = "Praise me, the almightly atomized Radioactive Man!", bad_excuse = "These atoms are too dense!")
+
+  puts "The danger level appears to be #{danger_level}!"
+if danger_level >= 50
+  puts bad_excuse
+
+elsif danger_level < 50 and danger_level > 10
```

Comment:
I am primarily a FE dev, so I still see new stuff in Ruby all the time, but I am unfamiliar with `and` being spelled out like that. Totally looks like it's fine, but I may encourage using `&&` as a default.

Reply:
Also, order matters with conditionals- think about saving the most complex logic for your `else` block, when that works.